### PR TITLE
add staked lp to prop tests

### PR DESCRIPTION
**Pull request by markonmars**

### CODE REVIEW

File: packages/health-computer/tests/tests/helpers/prop_test_strategies.rs [hunk 1]
```diff
@@ -273,8 +273,9 @@ pub fn random_health_computer() -> impl Strategy<Value = HealthComputer> {
             random_debts(denoms_data.clone()),
             random_coins(denoms_data.clone()),
             random_vault_positions(vaults_data.clone()),
+            random_coins(denoms_data.clone()),
         )
-            .prop_map(move |(kind, deposits, debts, lends, vaults)| HealthComputer {
+            .prop_map(move |(kind, deposits, debts, lends, vaults, staked_astro_lps)| HealthComputer {
```
Comment (piobab) [inline packages/health-computer/tests/tests/helpers/prop_test_strategies.rs:278]: Maybe we should generate with proper LP name structure for astro?

Comment (bobthebuidlr) [reply packages/health-computer/tests/tests/helpers/prop_test_strategies.rs:278]: Agree. I think we shouldn't use the same denoms for lp positions.

Comment (markonmars) [reply packages/health-computer/tests/tests/helpers/prop_test_strategies.rs:295]: added denom generation that matches astroport lp share tokens format

**Comment by piobab** [comment]
@markonmars I pushed commit fixing fmt issue

**Review by piobab — APPROVED** [review]

**Review by bobthebuidlr — APPROVED** [review]

**Pull request merged**